autogpts/autogpt/autogpt/logs/config.py

```diff
@@ -184,7 +190,7 @@ def configure_logging(
     json_logger.propagate = False
 
     # Disable debug logging from OpenAI library
-    openai_logger.setLevel(logging.WARNING)
+    openai_logger.setLevel(logging.INFO)
```

Comment:
Why? We don't want all of OpenAI's internal logging to show up in the application log, because it's quite a lot.